parsec/backend/cli/run.py

```diff
@@ -227,6 +227,26 @@ def handle_parse_result(self, ctx, opts, args):
     envvar="PARSEC_DB_MAX_CONNECTIONS",
     help="Maximum number of connections to the database if using PostgreSQL",
 )
+@click.option(
+    "--db-first-tries-number",
+    default=1,
+    show_default=True,
+    envvar="PARSEC_DB_FIRST_TRIES_NUMBER",
+    help=(
+        "Number of first tries allowed to connect to the database if using PostgreSQL, "
```

Comment:
Given PostgreSQL is the only "real" choice, I guess we can skip the "if using PostgreSQL" part and just say `Number of tries allowed during initial database connection (0 is unlimited)` and `Number of second waited between tries during initial database connection`

Reply:
Done